### ISSUE TITLE
Highlight branches on hover in Graph view

### ISSUE THREAD

**Issue by Surendrajat** [radareorg/cutter#2235]
**Environment information**
* Operating System: elementary 5.1.5
* Cutter version: 1.10.3 (built from latest source actually, tbh)
* File format: elf64
 * Arch: x86
 * Type: EXEC (Executable file)

**Describe the bug**
In the graph view, when following branches if there are multiple long branches together highlighting the color on hover is absent.

**To Reproduce**
1. Go to 'Graph view/widget/window'
2. Hover/Click on 'a branch/call/jump'

**Expected behavior**
The branch could be highlighted as in the attached screenshot. **Better**: When clicking on a branch keep it highlighted.

**Screenshots**
![graph-branch-highlight](https://user-images.githubusercontent.com/11215000/83935949-37db8b00-a7dc-11ea-8232-9ae728af9bfd.gif)

**Additional context**
Meh :P

**Comment by Surendrajat** [comment]
Moreover, if a single-click on the branch keeps the selected branch highlighted and a double-click on the branch brings you to the target block(just like a single-click on JEB), it will be nicer.
JEB ss:
![jeb_graph](https://user-images.githubusercontent.com/11215000/104798291-7eba4600-57eb-11eb-9e3c-9c18c9e51f16.gif)

**Comment by Petross404** [comment]
Excellent idea and very useful. But some users (me included) use the single click to act as double click on Kde for example. 

How will this feature respond?

**Comment by Surendrajat** [comment]
@Petross404 do you think that setting will affect all the apps? I don't use KDE but I think app-specific behavior can be defined within the app itself.

**Comment by Petross404** [comment]
> @Petross404 do you think that setting will affect all the apps? I don't use KDE but I think app-specific behavior can be defined within the app itself.

I am curious too. I think you are right about defining the behavior in Cutter itself.

**Comment by karliss** [comment]
@Petross404 Are you referring to the same option which controls if folders are opened by single or double click? Otherwise please clarify the exact option you are referring to. That shouldn't affect basic mouse event handling, otherwise a lot of programs would be broken. That option might affect  select/activated events for list views, which according to docs varies between OS so maybe also KDE config.

**Comment by Petross404** [comment]
> @Petross404 Are you referring to the same option which controls if folders are opened by single or double click? 

I am, yes. Is it irrelevant?

**Comment by Surendrajat** [comment]
> Is it irrelevant?

Yes. It only affects the file manager(s) afaik. Even if it does affect some other default KDE app, it shouldn't affect anything else.

**Comment by Petross404** [comment]
My mistake, sorry.
